armi/reactor/blocks.py

```diff
@@ -269,28 +269,16 @@ def getSmearDensity(self, cold=True):
 
         return smearDensity
 
-    def autoCreateSpatialGrids(self, systemSpatialGrid=None):
+    def autoCreateSpatialGrids(self, *args, **kwargs):
         """
         Creates a spatialGrid for a Block.
 
         Blocks do not always have a spatialGrid from Blueprints, but some Blocks can have their
         spatialGrids inferred based on the multiplicity of their components. This would add the
         ability to create a spatialGrid for a Block and give its children the corresponding
         spatialLocators if certain conditions are met.
-
-        Parameters
-        ----------
-        systemSpatialGrid : Grid, optional
-            Spatial Grid of the system-level parent of this Assembly that contains this Block.
-
-        Raises
-        ------
-        ValueError
-            If the multiplicities of the block are not only 1 or N or if generated ringNumber leads
-            to more positions than necessary.
         """
-        if self.spatialGrid is None:
-            self.spatialGrid = systemSpatialGrid
```

Comment:
I removed this action in favor of a `pass`. I'm not sure why it ever made sense to give a block the spatial grid of the core.

The `HexBlock` has a more specific implementation of this method, so for those blocks, something real will happen.